flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/InternalTimeServiceManager.java

```diff
@@ -52,7 +52,7 @@
 
 	private final ProcessingTimeService processingTimeService;
 
-	private final Map<String, HeapInternalTimerService<K, N>> timerServices;
+	public final Map<String, HeapInternalTimerService<K, N>> timerServices;
```

Comment:
Is this only public for testing?

Reply:
no, this is a mistake that should be reverted.